<launch>
    <node pkg="norlab_controllers_wrapper" exec="controller_node.py" name="controller_node" output="screen">
        <remap from="odom_in" to="icp_odom"/>
        <remap from="cmd_vel_out" to="nav_vel"/>
        <param name="controller_config" value="$(find-pkg-share norlab_controllers_wrapper)/params/marmotte/differential-orthexp.yaml"/>
        <param name="rotation_controller_config" value="$(find-pkg-share norlab_controllers_wrapper)/params/marmotte/snow/rotation-p.yaml"/>
    </node>
</launch>
